BigQuery/src/BigQueryClient.php

```diff
@@ -856,6 +856,29 @@ public function geography($value)
         return new Geography($value);
     }
 
+    /**
+     * Create a Json object.
```

Comment:
Do you think renaming this as BigQuery json will better?
```suggestion
     * Create a BigQuery Json object.
```

Reply:
Yeah good idea

Reply:
Done